### DISCUSSION TITLE
Question on Enhanced Exercise

### DISCUSSION THREAD

**Discussion by nmxdaven** [jgyates/genmon#1075] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

1.19.02

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

First off thanks so much for developing this. Its great. 

I can't seem to find any through documentation on the enhanced exercise add on in the wiki or web. It looks like it has built in descriptions for the exercise type but its a bit broken and only displays quiet. Is there documentation on the options more through than what is found in the wiki somewhere I am missing?

Thanks!

**Comment by jgyates** [comment, marked as answer]
The exercise add on does not replace the built in exercise cycle, it adds a 2nd exercise cycle. the add on starts the generator with the remote start commands 

The wiki entry for this add on is located [here](https://github.com/jgyates/genmon/wiki/1----Software-Overview#genexercisepy-optional). There is a link to the wiki page on the add on page on the Enhanced Exercise tile before enabling it.
